<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>911</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>AddTest</string>
    </attribute>
    <widget class="QPushButton" name="addQuestionButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>281</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить вопрос</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="testEdit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>471</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Название теста</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="testAddTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>1041</width>
       <height>781</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>344</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Тип теста (выбор/ответ)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Вопрос</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Варианты ответа (если нужны).</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Правильный ответ</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Instructions</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
